Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:bionic
+MAINTAINER devops@edx.org
+
+RUN apt-get update && apt-get upgrade -qy && apt-get install language-pack-en locales git python2.7 python-pip libmysqlclient-dev -qy && \
+pip install --upgrade pip setuptools && \
+rm -rf /var/lib/apt/lists/*
+
+RUN locale-gen en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+ENV EDXNOTES_CONFIG_ROOT /edx/etc
+ENV DJANGO_SETTINGS_MODULE notesserver.settings.yaml_config
+
+EXPOSE 8120
+RUN useradd -m --shell /bin/false app
+
+WORKDIR /edx/app/notes
+
+# Copy the requirements explicitly even though we copy everything below
+# this prevents the image cache from busting unless the dependencies have changed.
+COPY requirements/base.txt /edx/app/notes/requirements/base.txt
+
+# Dependencies are installed as root so they cannot be modified by the application user.
+RUN pip install -r requirements/base.txt
+
+RUN mkdir -p /edx/var/log
+
+# Code is owned by root so it cannot be modified by the application user.
+# So we copy it before changing users.
```

Comment:
Where is this copy cmd?

Reply:
Moved it - ty

Reply:
I verified file ownership is still root, the user directive is not taking place until run time, files are still owned by root despite the reorder.

Reply:
Can we update/move the comment so it isn't as confusing